.github/actions/plugin-build/action.yml

```diff
@@ -6,7 +6,7 @@ runs:
   using: 'composite'
   steps:
     - name: Cache assets directory
-      uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 #v4.0.2
+      uses: actions/cache@v4
```

Comment:
I made this change because GHA was failing with an error:

> Error: This request has been automatically failed because it uses a deprecated version of `actions/cache: 0c45773b623bea8c8e75f6c82b208c3cf94ea4f9`. Please update your workflow to use v3/v4 of actions/cache to avoid interruptions. Learn more: https://github.blog/changelog/2024-12-05-notice-of-upcoming-releases-and-breaking-changes-for-github-actions/#actions-cache-v1-v2-and-actions-toolkit-cache-package-closing-down

I'm confused why this is happening though because https://github.com/actions/cache/commit/0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 is [v4.0.2](https://github.com/actions/cache/releases/tag/v4.0.2) so it is already v4.

Reply:
I'm also confused why Dependabot didn't update this automatically. It updated some of the workflows but not all of them: https://github.com/ampproject/amp-wp/pull/7991

Reply:
Actually we need to define the paths for @dependabot to look for workflow files in `.github/actions/*`

```
  - package-ecosystem: github-actions
    directory: '.github/actions/setup-bun'
    schedule:
      interval: weekly
      day: saturday
    groups:
      actions:
        patterns:
          - 'actions/*'
    labels:
      - 'github-actions'
      - 'dependencies'
```

Reply:
Why wouldn't `directory: "/"` be handling this?

https://github.com/ampproject/amp-wp/blob/a9654fe1b144fa6e486b29281cdf3bb7ca0235ce/.github/dependabot.yml#L34-L40